frontend/public/components/utils/details-page.tsx

```diff
@@ -179,7 +179,19 @@ export const ResourcePodCount: React.SFC<ResourcePodCountProps> = ({ resource })
         path="spec.replicas"
         defaultValue="0"
       />
-    </dl>
+    </>
+  );
+};
+
+export const RuntimeClass: React.FC<RuntimeClassProps> = ({ resource, pathToClassName }) => {
```

Comment:
I'd just call this `path` to be consistent with `DetailsItem` and to avoid confusion with CSS `className` properties.

```suggestion
export const RuntimeClass: React.FC<RuntimeClassProps> = ({ resource, path }) => {
```

Reply:
I'd also have a preference for calling the prop `obj` instead of `resource` to be consistent with `DetailsItem`. `obj` also is more clearly a specific instance whereas `resource` sometimes refers to a type of resource in k8s.

Reply:
fixed